dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Register/Phone.cshtml.cs

```diff
@@ -35,7 +36,7 @@ public async Task<IActionResult> OnPost()
             return this.PageWithErrors();
         }
 
-        var pinGenerationResult = await _userVerificationService.GenerateSmsPin(MobileNumber!);
+        var pinGenerationResult = await _userVerificationService.GenerateSmsPin(PhoneHelper.FormatMobileNumber(MobileNumber!));
```

Comment:
If we're storing the formatted number in the DB. We should check that we can send texts to the formatted number